pandas/tests/arrays/categorical/test_operators.py

```diff
@@ -113,9 +114,34 @@ def test_comparisons(self):
         res = cat_rev > "b"
         tm.assert_numpy_array_equal(res, exp)
 
+        # check that zero-dim array gets unboxed
+        res = cat_rev > np.array("b")
+        tm.assert_numpy_array_equal(res, exp)
+
 
 class TestCategoricalOps(object):
 
+    def test_compare_frame(self):
+        # GH#24282 check that Categorical.__cmp__(DataFrame) defers to frame
+        data = ["a", "b", 2, "a"]
+        cat = Categorical(data)
+
+        df = DataFrame(cat)
+
+        for op in [operator.eq, operator.ne, operator.ge,
```

Comment:
we have a fixture (not sure if u r using it elsewhere)

Reply:
Doesn’t make sense to use In this context